api/serverless.yml

```diff
@@ -59,6 +59,16 @@ functions:
                 paths: 
                   userId: true
 
+  get-users:
+    handler: src/handlers/users.getAllUsers
+    memorySize: 128
+    description: "Gets all user information"
+    events:
+      - http:
+            path: /users
```

Comment:
shouldn't they have different paths

Reply:
it can have the same path for different requests. `GET /users vs. POST /users`